fxrunner/src/lib/osapi/handle.rs

```diff
@@ -73,3 +88,11 @@ impl TryFrom<HANDLE> for Handle {
         }
     }
 }
+
+/// Return the current module handle.
+pub fn get_module_handle() -> Result<ModuleHandle, io::Error> {
+    let mut handle = ModuleHandle::null();
+    check_nonzero(unsafe { libloaderapi::GetModuleHandleExA(0, null(), handle.as_out_ptr()) })?;
```

Comment:
Nit: is there a reason we're using `GetModuleHandleEx` (without `GET_MODULE_HANDLE_EX_FLAG_UNCHANGED_REFCOUNT`) and not `GetModuleHandle` or `GET_MODULE_HANDLE_EX_FLAG_UNCHANGED_REFCOUNT` in order to get a reference to the current module which we don't need to free? I know there's typically a risk with an non-refcounted module reference of another thread freeing the last handle, but given that we're just grabbing a handle to our current module that's not going to happen. In any case, it's not really all that important, but it's just a bit of extra code and I wasn't sure if there was a concrete reason we needed it.